<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>281</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>171</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Face</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>161</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Brightness</string>
      </property>
      <widget class="QSlider" name="sliderFaceBrightnessAlpha">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>160</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="sliderPosition">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>54</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Alpha 1.0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>54</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Beta</string>
       </property>
      </widget>
      <widget class="QSlider" name="sliderFaceBrightnessBeta">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>160</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Iris</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>161</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Brightness</string>
      </property>
      <widget class="QSlider" name="sliderIrisBrightnessAlpha">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>160</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="sliderPosition">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>54</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Alpha 1.0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>54</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Beta</string>
       </property>
      </widget>
      <widget class="QSlider" name="sliderIrisBrightnessBeta">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>160</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Estimation</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxFixRef">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>161</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Fixed Reference Point</string>
      </property>
      <widget class="QRadioButton" name="radioButtonLEC">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Left Eye Corner</string>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="radioButtonREC">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Right Eye Corner</string>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="radioButtonLWC">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Left Window Corner</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="radioButtonRWC">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Right Window Corner</string>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="radioButtonLIC">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Left Image Corner</string>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QSlider" name="sliderRavgSize">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>16</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
      <property name="sliderPosition">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RAVG Size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Polynomial degree</string>
      </property>
     </widget>
     <widget class="QSlider" name="sliderPolyDeg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
      <property name="sliderPosition">
       <number>4</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelVideoOutput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelEyeOutput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPlotOutput">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>450</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textOut">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>290</y>
      <width>451</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigura_es">
    <property name="title">
     <string>Configurações</string>
    </property>
    <addaction name="actionPasta_origem"/>
    <addaction name="actionPasta_destino"/>
    <addaction name="actionNome_de_arquivo"/>
   </widget>
   <addaction name="menuConfigura_es"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionListaArquivos"/>
   <addaction name="actionLocateEyeFeatures"/>
   <addaction name="actionEstimatePOR"/>
   <addaction name="separator"/>
   <addaction name="actionCancelar"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPasta_origem">
   <property name="text">
    <string>Pasta origem</string>
   </property>
  </action>
  <action name="actionPasta_destino">
   <property name="text">
    <string>Pasta destino</string>
   </property>
  </action>
  <action name="actionNome_de_arquivo">
   <property name="text">
    <string>Nome de arquivo</string>
   </property>
  </action>
  <action name="actionListaArquivos">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../usr/share/pixmaps/gnome-folder.png</normaloff>../../../../../../usr/share/pixmaps/gnome-folder.png</iconset>
   </property>
   <property name="text">
    <string>Lista Arquivos</string>
   </property>
   <property name="toolTip">
    <string>Lê a pasta de origem por arquivos a serem processados.</string>
   </property>
  </action>
  <action name="actionLocateEyeFeatures">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../usr/share/pixmaps/mysql-admin.xpm</normaloff>../../../../../../usr/share/pixmaps/mysql-admin.xpm</iconset>
   </property>
   <property name="text">
    <string>Localiza olho</string>
   </property>
   <property name="toolTip">
    <string>Localiza os pontos relevantes do olho esquerdo</string>
   </property>
  </action>
  <action name="actionEstimatePOR">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../usr/share/icons/cab_view.png</normaloff>../../../../../../usr/share/icons/cab_view.png</iconset>
   </property>
   <property name="text">
    <string>Estima o POR (Point-Of-Regard)</string>
   </property>
   <property name="toolTip">
    <string>Estima a posição da tela onde o usuário está olhando, com base na informação detectada dos olhos.</string>
   </property>
  </action>
  <action name="actionCancelar">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../usr/share/icons/default.kde4/32x32/actions/application-exit.png</normaloff>../../../../../../usr/share/icons/default.kde4/32x32/actions/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="toolTip">
    <string>Cancela a ação em curso.</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
